Resources/config/services.xml

```diff
@@ -9,6 +9,7 @@
         <parameter key="valiton_payment_saferpay.client.class">Valiton\Payment\SaferpayBundle\Client\Client</parameter>
         <parameter key="valiton_payment_saferpay.saferpay_plugin.class">Valiton\Payment\SaferpayBundle\Plugin\SaferpayPlugin</parameter>
         <parameter key="valiton_payment_saferpay.guzzle_client.class">Payment\HttpClient\GuzzleClient</parameter>
+	    <parameter key="payment.form.valiton_payment_saferpay_type.class">Valiton\Payment\SaferpayBundle\Form\SaferpayType</parameter>
```

Comment:
Code style: please use 4 spaces for a tab and correct indentation. And please keep to the bundle namespaces (valiton_payment_saferpay.form ...)